packages/ui/src/helpers/accountSettings/validators/util.ts

```diff
@@ -0,0 +1,56 @@
+import { FieldValidator, ValidationMode } from '../../../types';
+
+/**
+ * `shouldValidate` determines whether validator should be run, based on validation mode,
+ * input event type, and whether it has been blurred yet.
+ */
+export const shouldValidate = (
+  validationMode: ValidationMode,
+  eventType: 'change' | 'blur',
```

Comment:
nit: Can we use `FormEventType` here?

Reply:
Thanks for the catch again, adjusting.

Reply:
Fixed in [c0403a1](https://github.com/aws-amplify/amplify-ui/pull/2900/commits/c0403a12022bdfd6f63e8c6e9bd34000fc351831)!

Reply:
Thinking it should maybe named `InputEventType` instead of `FormEventType` to be clearer too... 🤔

Reply:
Above renaming done in dc6411f0f28c0d0a653839288c74031f5b33d10d